src/frontend/js/components/SearchFilterGroupModal/_styles.scss

```
@@ -5,6 +5,23 @@
   justify-content: space-between;
   max-width: 30rem;
 
+  &__close {
+    width: 1rem;
+    height: 1rem;
+    position: absolute;
+    top: 1.25rem;
+    right: 1rem;
+    background: transparent;
+    border: none;
+    padding: 0;
+
+    &__icon {
+      width: 1rem;
+      height: 1rem;
+      fill: r-theme-val(search-filters-group-modal, base-color);
+    }
+  }
```

If you use `.modal_closeButton` this code could be simplified :

```suggestion
 &__close {
    position: absolute;
    top: 0;
    right: 0;
  }
```